spec/spec_helper_acceptance_local.rb

```diff
@@ -37,6 +37,8 @@ def print_parsing_errors
     LitmusHelper.instance.run_shell('setenforce 0', expect_failures: true) if %r{redhat|oracle}.match?(os[:family])
 
     LitmusHelper.instance.run_shell('puppet module install stahnma/epel')
+
+    LitmusHelper.instance.run_shell('yum-config-manager --disable rhui-rhel-7-server-rhui-debug-rpms') if os[:family] == 'redhat' && os[:release].to_i == 7
```

Comment:
Looks like this fails on Scientific. I think you need to check if the os name is RH because RHUI is a RHEL thing.